assets/js/components/events/EventsTablePaginated.jsx

```diff
@@ -174,8 +174,8 @@ const query = gql`
 
 // TODO create event fragment?
```

Comment:
we're now using the event fields in a few places. could be worth looking into creating a fragment for it: https://www.apollographql.com/docs/react/advanced/fragments.html